From cc1b9c86d1b563318a2b9489b062f9519ae08b20 Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Wed, 23 Apr 2014 20:13:19 +0300
Subject: [PATCH] Fix file-roller>=3.12.1 detection (bug #10826)

File Roller 3.12.1 changed the name of its .desktop file from
file-roller.desktop to org.gnome.FileRoller.desktop.

We install a new symlink to support the new .desktop file name.
---
 scripts/Makefile.am | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index 4da529c..3804605 100644
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -5,12 +5,15 @@ wrapper_SCRIPTS =							\
 	ark.tap								\
 	file-roller.tap
 
-# Install symlink to 'gnome-file-roller.tap'
+# Install symlink to 'gnome-file-roller.tap' and 'org.gnome.FileRoller.tap'
 install-exec-hook:
 	$(mkinstalldirs) $(DESTDIR)$(wrapperdir)
 	-( cd $(DESTDIR)$(wrapperdir) ; \
 		test -f gnome-file-roller.tap \
 		|| ln -sf file-roller.tap gnome-file-roller.tap )
+	-( cd $(DESTDIR)$(wrapperdir) ; \
+		test -f org.gnome.FileRoller.tap \
+		|| ln -sf file-roller.tap org.gnome.FileRoller.tap )
 
 EXTRA_DIST =								\
 	$(wrapper_SCRIPTS)						\
-- 
1.9.2

